2021-02-05 18:41:01,415 INFO    MainThread:150452 [wandb_setup.py:_flush():70] setting env: {}
2021-02-05 18:41:01,415 INFO    MainThread:150452 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-02-05 18:41:01,415 INFO    MainThread:150452 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-05 18:41:01,415 INFO    MainThread:150452 [wandb_init.py:_log_setup():318] Logging user logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210205_184101-3sr6c9md\logs\debug.log
2021-02-05 18:41:01,415 INFO    MainThread:150452 [wandb_init.py:_log_setup():319] Logging internal logs to C:\Users\Juhyeon\Desktop\repository\Proxy-Anchor-CVPR2020\code\wandb\run-20210205_184101-3sr6c9md\logs\debug-internal.log
2021-02-05 18:41:03,235 INFO    MainThread:150452 [wandb_run.py:_console_start():1399] atexit reg
2021-02-05 18:41:03,236 INFO    MainThread:150452 [wandb_run.py:_redirect():1262] redirect: SettingsConsole.WRAP
2021-02-05 18:41:03,237 INFO    MainThread:150452 [wandb_run.py:_redirect():1297] Wrapping output streams.
2021-02-05 18:41:03,237 INFO    MainThread:150452 [wandb_run.py:_redirect():1313] Redirects installed.
2021-02-05 18:41:03,238 INFO    MainThread:150452 [wandb_run.py:_config_callback():663] config_cb None None {'LOG_DIR': '../logs', 'dataset': 'cars', 'sz_embedding': 512, 'sz_batch': 128, 'nb_epochs': 60, 'gpu_id': 0, 'nb_workers': 0, 'model': 'googlenet', 'loss': 'Triplet', 'optimizer': 'adamw', 'lr': 0.0001, 'weight_decay': 0.0001, 'lr_decay_step': 20, 'lr_decay_gamma': 0.5, 'alpha': 32, 'mrg': 0.1, 'IPC': None, 'warm': 1, 'bn_freeze': 1, 'l2_norm': 1, 'remark': ''}
2021-02-05 22:51:19,620 INFO    MainThread:150452 [wandb_run.py:_atexit_cleanup():1369] got exitcode: 1
2021-02-05 22:51:19,728 INFO    MainThread:150452 [wandb_run.py:_restore():1341] restore
2021-02-05 22:51:20,039 INFO    MainThread:150452 [wandb_run.py:_wait_for_finish():1492] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1274
  total_bytes: 1274
}

2021-02-05 22:51:22,044 INFO    MainThread:150452 [wandb_run.py:_wait_for_finish():1492] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7415
  total_bytes: 7415
}

2021-02-05 22:51:23,352 INFO    MainThread:150452 [wandb_run.py:_show_files():1714] logging synced files
2021-02-05 22:51:23,408 INFO    MainThread:150452 [internal.py:handle_exit():156] Internal process exited
